### DISCUSSION TITLE
Deploying the same application into multiple namespaces

### DISCUSSION THREAD

**Discussion by o4rz3l** [helmfile/helmfile#799] [Questions] [answered]
Hi, 
i'm exploring the usage of helmfile for our use cases. One of them is to deploy an application (jenkins) into multiple namespaces (per team).
I have looked trough the docs but did not finf this use case. I tried to expand the "namespace" value to be a list, but lintig tells me it should be a string.
Is this use case possible with a helmfile?

Thanks in advance!
Best regards, 
Oli

**Comment by yxxhero** [comment]
@ o4rz3l could you use multiple releases with different namespaces?

**Reply by o4rz3l** [reply]
hm... no. I'm talking about >400 releases. >80% share the same config. the rest would get different releases depending on their values.

**Comment by yxxhero** [comment, marked as answer]
@o4rz3l maybe you can use the go template syntax: `range`?

**Reply by o4rz3l** [reply]
hi @yxxhero , yes with `range` it is possible. I will see, whether it will work for the complete use-case. Thanks for your help!

**Reply by yxxhero** [reply]
@o4rz3l if you have more issues. please free to post new issues.
